files/SZWESAMP/ZWEIMVS2

```diff
@@ -0,0 +1,45 @@
+//ZWEIMVS2 JOB                                     
+//*                                                                    
+//* This program and the accompanying materials are made available     
+//* under the terms of the Eclipse Public License v2.0 which          
+//* accompanies this distribution, and is available at                 
+//* https://www.eclipse.org/legal/epl-v20.html                         
+//*                                                                    
+//* SPDX-License-Identifier: EPL-2.0                                   
+//*                                                                    
+//* Copyright Contributors to the Zowe Project. 2020, 2020             
+//*                                                                    
+//********************************************************************* 
+//*
+//* This job is used to create the APF load library for an instance
+//* Of Zowe. It is not needed if your choosen value of
+//* 'zowe.setup.dataset.authLoadlib' is equal to
+//* 'zowe.setup.prefix' + 'SZWELOAD'.
+//*
+//* When running this job, you should also run ZwEIMVS
+//*
+//********************************************************************* 
+//MKPDSE EXEC PGM=IKJEFT01                       
+//SYSTSPRT DD SYSOUT=A                            
+//SYSTSIN DD *                                    
+ALLOC NEW DA('{zowe.setup.dataset.authLoadlib}') +       
+dsntype(library) dsorg(po) recfm(u) lrecl(0) +    
+blksize(32760) unit(sysallda) space(30,15) tracks 
+//*                                               
+//AUTHCPY EXEC PGM=BPXBATCH                       
+//BPXPRINT DD SYSOUT=*                            
+//STDOUT   DD SYSOUT=*                            
+//STDERR   DD SYSOUT=*                            
+//STDPARM DD *                                    
+SH cd {zowe.runtimeDirectory} &&                                    
+cd components/zss &&    
+cp LOADLIB/ZWESIS01                               
+"//'{zowe.setup.dataset.authLoadlib}(ZWESIS01)'" &&      
```

Comment:
Will fail for any dataset with `$`.

Reply:
There is a limitation for `BPXBATCH` in-stream data: "Records in in-stream data sets are concatenated with blanks as separator characters." Each command, each string must fit in 80 chars.
It will fit for "normal" configuration, but if you exceed in dsn name certain number of dollar signs, which must be escaped, you will not fit and you can't continue on the next line.

If this will cause any problems, here or in other JCL, we can always allocate library for shell with "wider" record format and use it in `STDPARM`.

Reply:
Done,